Okta.Xamarin/Okta.Xamarin.Android/Okta.Xamarin.Android.csproj

```diff
@@ -54,39 +89,65 @@
       <Link>OktaConfig.cs</Link>
     </Compile>
     <Compile Include="ActivityCustomUrlSchemeInterceptor.cs" />
-    <Compile Include="OktaConfig.Android.cs" />
+    <Compile Include="MainActivity.cs" />
     <Compile Include="OidcClient.Android.cs" />
+    <Compile Include="OktaConfig.Android.cs" />
     <Compile Include="Resources\Resource.designer.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="Keystore\okta.xamarin.android-test\okta.xamarin.android-test.keyInfo">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
+    <None Include="Keystore\okta.xamarin.android-test\okta.xamarin.android-test.keystore">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
+    <None Include="Keystore\okta.xamarin.android\okta.xamarin.android.keyInfo">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
+    <None Include="Keystore\okta.xamarin.android\okta.xamarin.android.keystore">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
     <None Include="Resources\AboutResources.txt" />
+    <None Include="Assets\AboutAssets.txt" />
+    <None Include="Properties\AndroidManifest.xml" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Xamarin.Android.Support.CustomTabs">
-      <Version>28.0.0.1</Version>
-    </PackageReference>
-    <PackageReference Include="Xamarin.Android.Support.v7.AppCompat" Version="28.0.0.1" />
+    <AndroidResource Include="Resources\layout\Tabbar.xml" />
```

Comment:
Since we do not have any native view yet, should we exclude these resources for now? Do we need them?

Reply:
These files are used by the Xamarin forms shell and would cause issues if removed.